src/cluster.rs

```diff
@@ -725,9 +725,8 @@ impl RoutingInfo {
             }
             b"XGROUP" | b"XINFO" => get_arg(&args, 2).and_then(RoutingInfo::for_key),
             b"XREAD" | b"XREADGROUP" => {
-                let streams_position = args
-                    .iter()
-                    .position(|a| *a == Value::Data(b"STREAMS".to_vec()))?;
+                let streams_command = Value::Data("STREAMS".into());
+                let streams_position = args.iter().position(|a| *a == streams_command)?;
```

Comment:
Shouldn't even need to allocate this once
```suggestion
                let streams_position = args.iter().position(|a| match a { Value::Data(a) => == "STREAMS", _ => false })?;
```

Reply:
True!

Reply:
```suggestion
                let streams_position = args.iter().position(|a| match a {
                    Value::Data(a) => a == b"STREAMS",
                    _ => false,
                })?;
```